src/bridge.rs

```diff
@@ -58,7 +58,7 @@ pub fn discover_upnp() -> ::std::result::Result<Vec<String>, ::ssdp::SSDPError>{
 ///             break;
 ///         },
 ///         // Prompt the user to press the link button
-///         Err(HueError::BridgeError{error: BridgeError::LinkButtonNotPressed, ..}) => {
+///         Err(HueError(HueErrorKind::BridgeError(_,_,BridgeError::LinkButtonNotPressed),_)) => {
```

Comment:
This would be very similar to what it was before if the `BridgeError` variant was a struct variant as stated further down.